docs/docs/guide/react/README.md

```diff
@@ -31,7 +31,7 @@ import {
 
 const Component = () => {
   // Get the user to log in before a space can be obtained.
-  const identity = useIdentity({ login: true });
+  const identity = useIdentity();
```

Comment:
Removing `{login: true}` here makes the rest of the page confusing. The next two paragraphs on the page explain what the `{login: true}` option does. Need to update that text in light of `login` now defaulting to `true`.

Reply:
Looks like I left the comment in the wrong spot! Good job finding the right spot @wittjosiah 😛